test/validation/api/classification/odp_classification_common.c

```diff
@@ -392,6 +420,22 @@ odp_packet_t create_packet(cls_packet_info_t pkt_info)
 			ODPH_ERR("odph_udp_tcp_chksum failed\n");
 			return ODP_PACKET_INVALID;
 		}
+	} else if (pkt_info.gtpu) {
+		udp->src_port = odp_cpu_to_be_16(CLS_DEFAULT_SPORT);
+		udp->dst_port = odp_cpu_to_be_16(ODP_GTPU_UDP_PORT);
+		udp->length = odp_cpu_to_be_16(payload_len + ODPH_UDPHDR_LEN);
+		udp->chksum = 0;
+		odp_packet_has_udp_set(pkt, 1);
+		hlen = (uint8_t *)odp_packet_l4_ptr(pkt, NULL);
+		gtpu = (odph_gtpuhdr_t *)(hlen + sizeof(odph_udphdr_t));
+		gtpu->teid = odp_cpu_to_be_32(0xdeadbeef);
+		gtpu->gtp_hdr_info = 0x30;
+		gtpu->msg_type = 1;
+		gtpu->plen = 0;
```

Comment:
Here we have more magic numbers that should be #defined and/or commented.

- 0x30 really means GTP version 3 without optional header parts.
- msg type 1 really means GTP echo request.
- I think payload lenght should be sizeof(cls_test_packet_t) because we store that in the payload.

Reply:
Ack, Will add comments